tools/run_tests/xds_k8s_test_driver/framework/xds_url_map_test_resources.py

```diff
@@ -209,8 +220,16 @@ def setup(self, test_case_classes: 'Iterable[XdsUrlMapTestCase]') -> None:
         # Health Checks
         self.td.create_health_check()
         # Backend Services
-        self.td.create_backend_service()
-        self.td.create_alternative_backend_service()
+        #
+        # The backend services are created with HTTP2 instead of GRPC (the
+        # default) to disable validate-for-proxyless, because the affinity
+        # settings are not accepted by RCTH yet.
+        #
+        # TODO: delete _BackendHTTP2 from the parameters, to use the default
```

Comment:
optional: create a tracking issue for yourself, so we won't forget to remove this parameter.

If this parameter is going to stay for a long time, we could create an Absl flag to tell whether to create backend service with validation or not. So, it can adapt while the RCTH rollout for staging and prod is in different stage.